git-keeper-client/gkeepclient/create_config.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2017 Thuy Dinh
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Provides the function for creating a new client.cfg file.
+"""
+
+from os.path import isfile, expanduser, isdir, join
+from gkeepclient.text_ui import confirmation
+from gkeepcore.system_commands import mkdir
+
+
+def create_config():
+    """
+    Prompts the user for configuration parameters and creates client.cfg
```

Comment:
For consistency with the rest of the codebase (and PEP 8), write function docstrings as commands rather than descriptions: "Prompt the user ... and create client.cfg"